src/main/java/org/wso2/carbon/module/swiftiso20022/models/mt103models/Entity.java

```diff
@@ -0,0 +1,12 @@
+package org.wso2.carbon.module.swiftiso20022.models.mt103models;
+
+import java.util.List;
+
+/**
+ * Class to represent entity in the request payload.
+ */
+public class Entity {
+    String option;
```

Comment:
Let's add access modifiers and getters, setters. Applicable for all the places

Reply:
Add class modifiers to all models in commit 4e687a247135e6bc9d43274b4f9f97756cf64480